### PR TITLE
MaksimTsedov Review

### PR DESCRIPTION
**Pull request by UMSF-K13-1-Samoylenko**

### CODE REVIEW

File: Task7_Console_UI/SequenceMenu.cs [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+namespace Task7_Console_UI
+{
+    using System;
+    using Task7_Lib;
+
+    /// <summary>
+    /// Sequence menu class
+    /// </summary>
+    public class SequenceMenu
```
Comment (MaksimTsedov) [inline Task7_Console_UI/SequenceMenu.cs]: For supporting classes it is common to declare tem with `static ` keyword.

File: Task7_Lib/FibbonachiInRangeSequence.cs [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+                {
+                    yield return x;
+                }
+                else
+                {
+                    yield break;
+                }
+
+                if (y <= this.rightBorder)
+                {
```
Comment (MaksimTsedov) [inline Task7_Lib/FibbonachiInRangeSequence.cs]: Code repeat.

File: Task7_Console_UI/SequenceMenu.cs [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+                    }
+                    else
+                    {
+                        this.Instruction();
+                    }
+
+                    break;
+                }
+
+                case 2:
```
Comment (MaksimTsedov) [inline Task7_Console_UI/SequenceMenu.cs]: Magic number.

File: Task7_Lib/FibbonachiInRangeSequence.cs (hunk 2 above)
Comment (UMSF-K13-1-Samoylenko) [reply Task7_Lib/FibbonachiInRangeSequence.cs]: Recently I used array with two elements for removing code repeating, but Alexander says that it is not good idea and x,y is better solution.

Comment (MaksimTsedov) [reply Task7_Lib/FibbonachiInRangeSequence.cs]: OK.

**Comment by MaksimTsedov** [comment]
All right, I don`t see flaws.

**Pull request merged**